src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -101,10 +101,10 @@ const BioAndChat = (props) => {
     <View style={styles.bioAndSendChatContainer(isAnonimity)}>
       <View style={styles.containerBio}>
         {bio === null || bio === undefined ? (
-          <Text style={styles.bioText(isAnonimity)}>Send a message</Text>
+          <Text style={styles.bioText}>Send a message</Text>
         ) : (
           <Pressable onPress={openBio}>
-            <Text linkStyle={styles.seeMore} style={styles.bioText(isAnonimity)}>
+            <Text linkStyle={styles.seeMore} style={styles.bioText}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `bioText` style no longer takes `isAnonimity` as a parameter. This could potentially change the appearance of the text depending on the value of `isAnonimity`. Please ensure that this change is intentional and does not negatively affect the UI.
```diff
- <Text style={styles.bioText(isAnonimity)}>Send a message</Text>
+ <Text style={styles.bioText}>Send a message</Text>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->